modal/cli/app.py

```diff
@@ -111,3 +111,60 @@ async def stop(
         app_id = await get_app_id_from_name.aio(name, env, client)
     req = api_pb2.AppStopRequest(app_id=app_id, source=api_pb2.APP_STOP_SOURCE_CLI)
     await client.stub.AppStop(req)
+
+
+@app_cli.command("history", no_args_is_help=True)
+@synchronizer.create_blocking
+async def history(
+    app_id: str = Argument("", help="Look up an App's deployment history by its ID"),
+    *,
+    env: Optional[str] = ENV_OPTION,
+    json: bool = False,
+):
+    """Show App Deployment history
+
+    **Examples:**
+
+    Get the history based on an app ID:
+
+    ```bash
+    modal app history ap-123456
+    ```
+
+    """
+    if not app_id:
+        raise UsageError("Must pass either an ID or a name.")
+
+    env = ensure_env(env)
+    client = await _Client.from_env()
+
+    resp: api_pb2.AppDeploymentHistoryResponse = await client.stub.AppDeploymentHistory(
+        api_pb2.AppDeploymentHistoryRequest(app_id=app_id)
+    )
+
+    columns: List[Union[Column, str]] = [
+        "id",
+        "Version",
+        "Deployed by",
+        "Deployed at",
+        "State",
```

Comment:
I don't think we need this either -- it will be `stopped` for every version aside from the latest one, and the state of the latest deployment is already shown in `modal app list`.

Reply:
Hm, what if you roll back to a previous version? I actually think the web page for this is confusing since it doesn't include which version is the one running, you just have to assume it's the top one

Reply:
Yeah it will always be the top one — the way rollbacks will work for us is that the version number will continue to increment and there will be some metadata indicating the state that we reset to. We’ll need to figure out how to enhance the table here when we add that metadata to the history response, ideally in a way that looks similar to what we do on the web.

Reply:
If you want to make the active version more salient here you could highlight it in green or bold white or something similar to what we do with `modal profile list`